src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form__search"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        class="input__big input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        name="serch__bar__big"
```

Comment:
6 point in checklist
use camelCase for values of name attribute
```suggestion
        name="searchBarBig"
```